CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/brackets/BracketHandlerPotion.java

```diff
@@ -34,9 +34,15 @@ public BracketHandlerPotion() {
     
     public static void rebuildRegistry() {
         potionNames.clear();
-        for(Potion potion : Potion.REGISTRY) {
-            potionNames.put(potion.getRegistryName().toString(), potion);
+        
+        Iterator<Potion> iterator = Potion.REGISTRY.iterator();
+        while(iterator.hasNext()) {
```

Comment:
Eclipse was whining about that method not working (though it did?) so I used an iterator instead...